Assets/Scripts/Steam/SteamManager.cs

```diff
@@ -63,6 +63,8 @@ protected static void SteamAPIDebugTextHook(int nSeverity, System.Text.StringBui
         HAuthTicket sessionTicketHandle = HAuthTicket.Invalid;
         string sessionTicketString = null;
 
+        Callback<GetTicketForWebApiResponse_t> m_AuthTicketForWebApiResponseCallback;
```

Comment:
While the rest of this file makes use in some cases of the prefix `m_`, adding _new_ things within this project we should make sure that new additions match the overall project style guidelines (at least in this case).